tests/test_handlers.rs

```diff
@@ -672,6 +672,6 @@ fn test_unsafe_path_route() {
     let bytes = srv.execute(response.body()).unwrap();
     assert_eq!(
         bytes,
-        Bytes::from_static(b"success: http:%2F%2Fexample.com")
```

Comment:
i think `.match_info()` should contain unquoted values

Reply:
Wouldn't it be better for `Params` to have method that can unquote me so that user would be able to unquote on demand?